pom.xml

```
@@ -21,6 +21,7 @@
     <compiler-plugin.version>3.8.1</compiler-plugin.version>
     <enforcer-plugin.version>3.4.1</enforcer-plugin.version>
     <assertj.version>3.24.2</assertj.version>
+    <opentelemetry-alpha.version>1.28.0-alpha</opentelemetry-alpha.version>
```

```suggestion
    <opentelemetry.version>1.28.0-alpha</opentelemetry.version>
```

And use the same under